src/qt/rpcconsole.cpp

```diff
@@ -463,7 +463,19 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
     ui->dataDir->setToolTip(ui->dataDir->toolTip().arg(QString(nonbreaking_hyphen) + "datadir"));
     ui->blocksDir->setToolTip(ui->blocksDir->toolTip().arg(QString(nonbreaking_hyphen) + "blocksdir"));
     ui->openDebugLogfileButton->setToolTip(ui->openDebugLogfileButton->toolTip().arg(PACKAGE_NAME));
-    ui->peerConnectionTypeLabel->setToolTip(ui->peerConnectionTypeLabel->toolTip().arg("addnode").arg(QString(nonbreaking_hyphen) + "addnode").arg(QString(nonbreaking_hyphen) + "connect"));
+    ui->peerConnectionTypeLabel->setToolTip(
+        ui->peerConnectionTypeLabel->toolTip()
+            .arg(tr("<ul>"
+                    "<li>Inbound Relay: initiated by peer</li>"
+                    "<li>Outbound Full Relay: default</li>"
+                    "<li>Outbound Block Relay: does not relay transactions or addresses</li>"
+                    "<li>Outbound Manual: added using RPC %1 or %2/%3 configuration options</li>"
+                    "<li>Outbound Feeler: short-lived, for testing addresses</li>"
+                    "<li>Outbound Address Fetch: short-lived, for soliciting addresses</li>"
+                    "</ul>")
```

Comment:
I'm not sure if the markup should be replaced by placeholders here. I see inline *.cpp markup elsewhere in the GUI codebase.

Reply:
resolved